src/modules/peek/Peek.FilePreviewer/FilePreview.xaml.cs

```diff
@@ -74,5 +88,11 @@ private async Task OnFilePropertyChanged()
                 PreviewSizeChanged?.Invoke(this, new PreviewSizeChangedArgs(new Size(1280, 720)));
             }
         }
+
+        private void PreviewBrowser_NavigationCompleted(WebView2 sender, Microsoft.Web.WebView2.Core.CoreWebView2NavigationCompletedEventArgs args)
```

Comment:
FYI: In my PR I rework how we trigger UI changes for control visibility so I think we won't need this event handler.